test/parallel/test-worker-cleanexit-with-js.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+require('../common');
+
+// Harden the thread interactions on the exit path.
+// Ensure workers are able to bail out safe at
+// arbitrary execution points. By using a number of
+// internal modules as load candidates, the expectation
+// is that those will be at various control flow points
+// preferrably in the JS land.
+
+const { Worker } = require('worker_threads');
+for (let i = 0; i < 10; i++) {
+  new Worker('for (let i = 0; i < 1000; i++) {' +
+      "require('v8').deserialize(require('v8').serialize({ foo: 'bar' }));" +
+      "require('vm').runInThisContext('x = \"foo\";');" +
+      "eval('const y = \"vm\";');" +
+      '}', { eval: true });
+}
+
+// Allow workers to go live.
+setTimeout(() => {
+  // process.exit(0);
```

Comment:
Maybe remove the commented out line?

Reply:
@Trott - it was a mistake to comment that out! The abrupt exit is required for the thread interactions to come in to play. I opened it up now.

Reply:
so no wonder why CI did not report a single related failure. Realistically I was expecting one or two that this test can throw, based on its original run that threw up a lot!